packages/matchbox/src/components/ThemeProvider/globalStyles.js

```diff
@@ -0,0 +1,141 @@
+import { tokens } from '@sparkpost/design-tokens';
+
+export default `
+  html {
+    position: relative;
+    box-sizing: border-box;
+    font-size: 16px;
```

Comment:
Like we were talking about in the change, should we use `$DEFAULT_FONT_SIZE`?

Reply:
We should, but the JS tokens don't export a default font size, just the scale. This is one token we should add though.

Reply:
I'll cut a ticket to add a new token